### PR TITLE
Update to new format

### PR DESCRIPTION
**Pull request by ludufre**
This update fix the missing "chat_list" of the new format.

### CODE REVIEW

**Comment by residentsummer** [comment]
Hi, @ludufre .

Thanks for taking time to fix this!

Reviewing [messages in gitter](https://gitter.im/residentsummer_watoi/Lobby?at=612861bad206b85e4f660f13) I think there is one more field needs to be fixed - `creation`. Could you please add it to the PR?

File: watoi/main.m [hunk 1]
```diff
@@ -425,7 +425,7 @@ - (NSString *) guessOurJID {
 }
 
 - (void) importMessages {
-    NSString *query = @"SELECT * FROM messages where"
+    NSString *query = @"SELECT * FROM legacy_available_messages_view where"
```
Comment (residentsummer) [inline watoi/main.m:428]: Why is that? As I can see in the [schema](https://github.com/andreas-mausch/whatsapp-viewer/blob/master/data/msgstore.db.schema.sql), this view adds some fields (unused by us) and does some kind of filtering on `messages` table. Do you know what's going on there?

**Review by residentsummer — CHANGES_REQUESTED** [review]
Reviewing messages [in gitter](https://gitter.im/residentsummer_watoi/Lobby?at=612861bad206b85e4f660f13) I think there is one more field needs to be fixed - creation. Could you please add it to the PR?

File: watoi/main.m (hunk 1 above)
Comment (ludufre) [reply watoi/main.m:428]: I have no idea. I just used that view because have most of the previous fields.

**Comment by ludufre** [comment]
Adjusted

> Reviewing messages [in gitter](https://gitter.im/residentsummer_watoi/Lobby?at=612861bad206b85e4f660f13) I think there is one more field needs to be fixed - creation. Could you please add it to the PR?

Ok, I commited. Thanks!

**Comment by amaanm** [comment]
I was receiving the `chat_list` error and can confirm I was just able to migrate my messages using this branch with WhatsApp v2.21.190.

Thanks @ludufre and @residentsummer!

File: watoi/main.m (hunk 1 above)
Comment (residentsummer) [reply watoi/main.m:428]: Ok, looks like this view handles "deleted" messages not being really deleted from msgstore. My only concern is that they may eventually drop this view... Will do for now.

**Review by residentsummer — APPROVED** [review]
LGTM

**Comment by residentsummer** [comment]
Thank you, @ludufre for this PR and @amaanm for testing!

**Pull request merged**